Add docs for setting custom tags for Argo CD (#21796)
DataDog/documentation: content/en/continuous_delivery/deployments/argocd.md

After this final step is completed, you can start monitoring your Argo CD deployments in Datadog.

## Adding custom tags to deployment executions

You can optionally add custom tags to the deployment executions generated from Argo CD applications deployments. These tags can be used to filter, group, and aggregate deployment executions in Datadog.

> To add custom tags, add the `dd_customtags` annotation to your Argo CD application annotations. The value of the annotation should be a comma-separated list of tags, structured as `key:value` pairs. For example:

Reviewer: ```suggestion
To add custom tags, add the `dd_customtags` annotation to your Argo CD application annotations and set the value to a comma-separated list of tags, structured as `key:value` pairs. For example:
```
